src/general_utils.erl

```diff
@@ -0,0 +1,74 @@
+%%%-------------------------------------------------------------------
+%%% @author kevin
```

Comment:
I suggest fixing this to be copyrighted by you and licensed by a Apache license, as Antidote.